云编译K2P时mtwifi报错

反馈bug/问题模板，提建议请删除

## 1.关于你要提交的问题

Q：是否搜索了issue (使用 "x" 选择)
* [x] 没有类似的issue

## 2. 详细叙述

### (1) 具体问题

A：使用GitHub云编译K2P固件，每次都在mt_wifi这里出错

### (2) 路由器型号和固件版本

A：K2P

### (3) 详细日志

> 2020-07-05T18:30:53.3729950Z echo "pdnsd-alt" >> /home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/target-mipsel_24kc_musl/pkginfo/pdnsd-alt.default.install
2020-07-05T18:30:53.3730283Z make[3]: Leaving directory '/home/runner/work/OpenWRT/OpenWRT/openwrt/package/lean/pdnsd-alt'
2020-07-05T18:30:53.3730593Z time: package/lean/pdnsd-alt/compile#9.56#1.65#12.95
2020-07-05T18:30:53.3730930Z make[3]: Entering directory '/home/runner/work/OpenWRT/OpenWRT/openwrt/package/lean/mt/drivers/mt_wifi'
2020-07-05T18:30:53.3731375Z touch /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.prepared_fadf52af00f13cba85d619cf70e1632e_6664517399ebbbc92a37c5bb081b5c53_check
2020-07-05T18:30:53.3731528Z true
2020-07-05T18:30:53.3731879Z [ ! -d ./src/ ] || cp -fpR ./src/. /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1
2020-07-05T18:30:53.3732313Z touch /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.prepared_fadf52af00f13cba85d619cf70e1632e_6664517399ebbbc92a37c5bb081b5c53
2020-07-05T18:30:53.3732677Z rm -f /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.configured_*
2020-07-05T18:30:53.3733035Z rm -f /home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/target-mipsel_24kc_musl/stamp/.mt_wifi_installed
2020-07-05T18:30:53.3737687Z (cd /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/./; if [ -x ./configure ]; then find /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/ -name config.guess | xargs -r chmod u+w; find /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/ -name config.guess | xargs -r -n1 cp --remove-destination /home/runner/work/OpenWRT/OpenWRT/openwrt/scripts/config.guess; find /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/ -name config.sub | xargs -r chmod u+w; find /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/ -name config.sub | xargs -r -n1 cp --remove-destination /home/runner/work/OpenWRT/OpenWRT/openwrt/scripts/config.sub; AR="mipsel-openwrt-linux-musl-gcc-ar" AS="mipsel-openwrt-linux-musl-gcc -c -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -ffile-prefix-map=/home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1=mt_wifi-1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro" LD=mipsel-openwrt-linux-musl-ld NM="mipsel-openwrt-linux-musl-gcc-nm" CC="mipsel-openwrt-linux-musl-gcc" GCC="mipsel-openwrt-linux-musl-gcc" CXX="mipsel-openwrt-linux-musl-g++" RANLIB="mipsel-openwrt-linux-musl-gcc-ranlib" STRIP=mipsel-openwrt-linux-musl-strip OBJCOPY=mipsel-openwrt-linux-musl-objcopy OBJDUMP=mipsel-openwrt-linux-musl-objdump SIZE=mipsel-openwrt-linux-musl-size CFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -mips16 -minterlink-mips16 -ffile-prefix-map=/home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1=mt_wifi-1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro " CXXFLAGS="-Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -msoft-float -mips16 -minterlink-mips16 -ffile-prefix-map=/home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1=mt_wifi-1 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro " CPPFLAGS="-I/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/target-mipsel_24kc_musl/usr/include -I/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/include -I/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include/fortify -I/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/include " LDFLAGS="-L/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/target-mipsel_24kc_musl/usr/lib -L/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/target-mipsel_24kc_musl/lib -L/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/usr/lib -L/home/runner/work/OpenWRT/OpenWRT/openwrt/staging_dir/toolchain-mipsel_24kc_gcc-8.4.0_musl/lib -znow -zrelro "   ./configure --target=mipsel-openwrt-linux --host=mipsel-openwrt-linux --build=x86_64-pc-linux-gnu --program-prefix="" --program-suffix="" --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --libexecdir=/usr/lib --sysconfdir=/etc --datadir=/usr/share --localstatedir=/var --mandir=/usr/man --infodir=/usr/info --disable-nls  ; fi; )
2020-07-05T18:30:53.3738926Z touch /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.configured_44131350ef320c1fe08400006e5845b8
2020-07-05T18:30:53.3739289Z rm -f /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.built
2020-07-05T18:30:53.3739653Z touch /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.built_check
2020-07-05T18:30:53.3740121Z touch /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.built
2020-07-05T18:30:53.3740619Z rm -rf /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi.installed /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi
2020-07-05T18:30:53.3741009Z mkdir -p /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi
2020-07-05T18:30:53.3741407Z install -d -m0755 /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi/etc/init.d/
2020-07-05T18:30:53.3741805Z install -d -m0755 /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi/lib/wifi/
2020-07-05T18:30:53.3742214Z install -d -m0755 /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi/etc/wireless/mt7615/
2020-07-05T18:30:53.3742796Z install -d -m0755 /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi/etc_ro/Wireless/RT2860AP/
2020-07-05T18:30:53.3743186Z install -d -m0755 /home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi/lib/preinit/
2020-07-05T18:30:53.3743322Z ifeq (y,y)
2020-07-05T18:30:53.3743604Z bash: -c: line 0: syntax error near unexpected token `y,y'
2020-07-05T18:30:53.3743840Z bash: -c: line 0: `ifeq (y,y)'
2020-07-05T18:30:53.3744482Z Makefile:74: recipe for target '/home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi.installed' failed
2020-07-05T18:30:53.3744900Z make[3]: *** [/home/runner/work/OpenWRT/OpenWRT/openwrt/build_dir/target-mipsel_24kc_musl/mt_wifi-1/.pkgdir/mt_wifi.installed] Error 1
2020-07-05T18:30:53.3745269Z make[3]: Leaving directory '/home/runner/work/OpenWRT/OpenWRT/openwrt/package/lean/mt/drivers/mt_wifi'
2020-07-05T18:30:53.3745429Z time: package/lean/mt/drivers/mt_wifi/compile#0.14#0.04#0.20
2020-07-05T18:30:53.3745815Z package/Makefile:111: recipe for target 'package/lean/mt/drivers/mt_wifi/compile' failed
2020-07-05T18:30:53.3746050Z make[2]: *** [package/lean/mt/drivers/mt_wifi/compile] Error 2
2020-07-05T18:30:53.3746200Z make[2]: *** Waiting for unfinished jobs....

--- a/package/lean/mt/drivers/mt_wifi/Makefile
+++ b/package/lean/mt/drivers/mt_wifi/Makefile
@@ -46,29 +46,26 @@ ifeq ($(CONFIG_MTK_CHIP_MT7603E_MT7612E),y)
 	$(INSTALL_BIN) ./files/7603_7612-l1profile.dat $(1)/etc/wireless/l1profile.dat
 	$(INSTALL_BIN) ./files/mt7603.dat $(1)/etc/wireless/mt7615/mt7615.1.dat
 	$(INSTALL_BIN) ./files/mt7612.dat $(1)/etc/wireless/mt7615/mt7615.2.dat
-	$(INSTALL_BIN) ./files/82_load_wifi.sh $(1)/lib/preinit/
 else ifeq ($(CONFIG_MTK_CHIP_MT7603E_MT7615E),y)
 	$(INSTALL_BIN) ./files/7603_7615-l1profile.dat $(1)/etc/wireless/l1profile.dat
 	$(INSTALL_BIN) ./files/mt7603.dat $(1)/etc/wireless/mt7615/mt7615.1.dat
 	$(INSTALL_BIN) ./files/mt7615.dat $(1)/etc/wireless/mt7615/mt7615.2.dat
-	$(INSTALL_BIN) ./files/82_load_wifi.sh $(1)/lib/preinit/
 else
-	ifeq ($(CONFIG_MTK_CHIP_MT7615E_DBDC),y)
-		$(INSTALL_BIN) ./files/dbdc.l1profile.dat $(1)/etc/wireless/l1profile.dat
-		$(INSTALL_BIN) ./files/mt7615.1.2G.dat $(1)/etc/wireless/mt7615/
-		$(INSTALL_BIN) ./files/mt7615.1.5G.dat $(1)/etc/wireless/mt7615/
-		$(INSTALL_BIN) ./files/82_load_wifi.sh $(1)/lib/preinit/
-	else
-		$(INSTALL_BIN) ./files/7615.l1profile.dat $(1)/etc/wireless/l1profile.dat
-		$(INSTALL_BIN) ./files/mt7615.2G.dat $(1)/etc/wireless/mt7615/mt7615.1.dat
-		$(INSTALL_BIN) ./files/mt7615.5G.dat $(1)/etc/wireless/mt7615/mt7615.2.dat
-		$(INSTALL_BIN) ./files/82_load_wifi.sh $(1)/lib/preinit/
-	endif
+ifeq ($(CONFIG_MTK_CHIP_MT7615E_DBDC),y)
+	$(INSTALL_BIN) ./files/dbdc.l1profile.dat $(1)/etc/wireless/l1profile.dat
+	$(INSTALL_BIN) ./files/mt7615.1.2G.dat $(1)/etc/wireless/mt7615/
+	$(INSTALL_BIN) ./files/mt7615.1.5G.dat $(1)/etc/wireless/mt7615/
+else
+	$(INSTALL_BIN) ./files/7615.l1profile.dat $(1)/etc/wireless/l1profile.dat
+	$(INSTALL_BIN) ./files/mt7615.2G.dat $(1)/etc/wireless/mt7615/mt7615.1.dat
+	$(INSTALL_BIN) ./files/mt7615.5G.dat $(1)/etc/wireless/mt7615/mt7615.2.dat
+endif
 endif
 	$(INSTALL_BIN) ./files/SingleSKU.dat $(1)/etc_ro/Wireless/RT2860AP/
 	$(INSTALL_BIN) ./files/SingleSKU_BF.dat $(1)/etc_ro/Wireless/RT2860AP/
 	$(INSTALL_BIN) ./files/mt7615.lua $(1)/lib/wifi
 	$(INSTALL_BIN) ./files/firmware.sh $(1)/etc/init.d/
+	$(INSTALL_BIN) ./files/82_load_wifi.sh $(1)/lib/preinit/
 endef
 
 $(eval $(call BuildPackage,mt_wifi))

--- a/target/linux/ramips/mt7621/base-files/etc/board.d/01_leds
+++ b/target/linux/ramips/mt7621/base-files/etc/board.d/01_leds
@@ -19,8 +19,8 @@ led_wlan="$(get_dt_led wlan)"
 case $board in
 d-team,newifi-d2)
 	ucidef_set_led_netdev "internet" "internet" "$boardname:amber:internet" "wan"
-	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "$boardname:blue:wlan2g" "wlan0"
-	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "$boardname:blue:wlan5g" "wlan1"
+	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "$boardname:blue:wlan2g" "ra0"
+	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "$boardname:blue:wlan5g" "rai0"
 	;;
 d-team,pbr-m1|\
 gehua,ghl-r-001)

